HeterogeneousCore/AlpakaTest/plugins/alpaka/TestAlgo.dev.cc

```diff
@@ -182,6 +182,24 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
         output[i] = {x, input[i].y(), input[i].z(), input[i].id(), input[i].flags(), input[i].m()};
       }
     }
+
+    template <typename TAcc, typename = std::enable_if_t<alpaka::isAccelerator<TAcc>>>
+    ALPAKA_FN_ACC void operator()(TAcc const& acc,
```

Comment:
(independently of this PR)

For kernels that are defined in `ALPAKA_ACCELERATOR_NAMESPACE`, I came to wonder the necessity or usefulness of defining these (member) functions as templates over `TAcc`. The kernel itself tends to have some kind of assumption on the dimensions, and the caller uses one of the `Acc<N>D` explicitly with `alpaka::exec()`. In this particular case the argument could be directly
```suggestion
    ALPAKA_FN_ACC void operator()(Acc1D const& acc,
```